openedx/features/course_card/tests/helpers.py

```diff
@@ -1,15 +1,28 @@
+"""
+Unit test helper file
+"""
 from datetime import datetime, timedelta
 
 from custom_settings.models import CustomSettings
-from lms.djangoapps.onboarding.models import UserExtendedProfile, EmailPreference, Organization
+from lms.djangoapps.onboarding.models import EmailPreference, Organization, UserExtendedProfile
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
-from student.tests.factories import UserFactory, CourseEnrollmentFactory
 from student.models import UserProfile
+from student.tests.factories import UserFactory
 
 from ..models import CourseCard
 
 
 def set_course_dates(course, enrollment_start, enrollment_end, course_start, course_end):
+    """
+    Args:
+        course (:obj:Course): Contains the course object
+        enrollment_start (:int): Contains the enrollment_start days
+        enrollment_end (:int): Contains the enrollment_end days
+        course_start (:int): Contains the course_start days
+        course_end (:int): Contains the course_end days
+    Returns:
+        A course (CourseOverview)
+    """
```

Comment:
Maybe we can also add a summary at the start of the docstrings, as mentioned in the recommendation, here and in other places